scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1304,6 +1306,47 @@ private void updateInheritedPropagatedVariables(Map<String, Serializable> propag
         }
     }
 
+    @Override
+    public boolean isWallTimeSet() {
```

Comment:
rename `isWallTimeSet` in `isWalltimeSet`

Reply:
not this one, it's an ovverride of an existing method which I don't want to modify